openapi.yml

```diff
@@ -300,21 +305,6 @@ paths:
           description: This region string is Fx domain language, and built from Fx expectations. Parameter values are not case sensitive. See [Firefox Home & New Tab Regional Differences](https://mozilla-hub.atlassian.net/wiki/spaces/FPS/pages/80448805/Regional+Differences).
           schema:
             type: string
-            enum: [
-                # relevant docs: https://docs.google.com/document/d/1omclr-eETJ7zAWTMI7mvvsc3_-ns2Iiho4jPEfrmZfo
-                US,
-                CA,
-                DE,
-                GB,
-                IE,
-                FR,
-                ES,
-                IT,
-                IN,
-                CH,
-                AT,
-                BE,
-              ]
```

Comment:
This is unrelated to time-to-read, but we just happened to notice that [#55](https://github.com/Pocket/firefox-api-proxy/pull/55) accidentally re-introduced this enum.